src/main/java/com/karanteam/doorkeeper/controller/AdminController.java

```diff
@@ -34,7 +39,12 @@ public AdminController(AdminService adminService, JwtService jwtService) {
     public ResponseEntity<Void> setCapacity(String xToken, CapacityBody capacityBody) {
         String userId = jwtService.parseToken(xToken, Role.EMPLOYEE, Role.HR);
         log.info("Received set capacity call with userId: " + userId);
-        adminService.setCapacity(capacityBody);
-        return ResponseEntity.ok().build();
+        try {
+            adminService.setCapacity(capacityBody);
+            officeMapService.recalculatePositions();
```

Comment:
I don't like this solution, but it would be a circular dependency if I would put it into admin service.